spec/display_board_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'stringio'
+
+def display_board(console, board)
+    $stdout = console
+    board.each do |row|
+        puts row.join(" ")
+    end
+end
+
+
+describe 'display_board' do
+    it 'displays the board given output console' do
+      console = StringIO.new
```

Comment:
You might find `let` useful, so that you don't have to repeat the line for creating `console` in each spec.  https://rspec.info/features/3-12/rspec-core/helper-methods/let/